src/components/aboutModal/aboutModal.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { AboutModal as PfAboutModal, TextContent, TextList, TextListItem } from '@patternfly/react-core';
+import { detect } from 'detect-browser';
+import moment from 'moment/moment';
+import { helpers } from '../../helpers';
+import { useUserApi } from '../../hooks/useLoginApi';
+import { useStatusApi, type ApiStatusSuccessType } from '../../hooks/useStatusApi';
+import backgroundImageSrc from '../../images/aboutBg.png';
+
+interface AboutModalProps {
+  currentYear?: string;
+  isOpen?: boolean;
+  onClose?: () => void;
+  titleImg?: string;
+  uiName?: string;
+  uiVersion?: string;
+  useStatus?: typeof useStatusApi;
+  useUser?: typeof useUserApi;
+}
+
+const AboutModal: React.FC<AboutModalProps> = ({
+  currentYear = moment.utc(helpers.getCurrentDate()).format('YYYY'),
+  isOpen = false,
+  onClose = Function.prototype,
+  titleImg = helpers.getTitleImg(),
+  uiName = helpers.UI_NAME,
+  uiVersion = helpers.UI_VERSION,
+  useStatus = useStatusApi,
+  useUser = useUserApi
+}) => {
+  const { t } = useTranslation();
+  const { getStatus } = useStatus();
+  const { getUser } = useUser();
+  const [userName, setUserName] = useState<string>();
+  const [stats, setStats] = useState<ApiStatusSuccessType>();
+  const browser = detect();
+  const loadingClassName = (!stats && 'fadein') || '';
+
+  useEffect(() => {
+    if (isOpen && !stats) {
+      getUser().then(username => setUserName(username));
+      getStatus().then(
+        data => setStats(data),
+        error => console.error(`About status error: ${error} `)
+      );
+    }
+  }, [isOpen, getStatus, getUser, stats]);
```

Comment:
Do we need to have stats in the dependency array here ? It's already used in the condition so I had the impression it didn't need to be in the array as well.

Reply:
good catch! you are correct we technically don't need it since it has the potential to create a doom loop... i'll remove it and eslint disable, currently, line `48`